### PR TITLE
issue-158-verify-auxiliary-cotains-class-definition

### PR DESCRIPTION
**Pull request by lumig242**
#158

### CODE REVIEW

File: assets/tests/CodeCheckerService.js [hunk 1]
```diff
@@ -25,10 +25,18 @@ tie.factory('CodeCheckerService', [
       return code.indexOf(functionDefinition) !== -1;
     };
 
+    var _checkIfClassDefinitionExistsInCode = function(className, code) {
+      var classDefinition = 'class ' + className + '(object):';
```
Comment (eyurko) [inline assets/tests/CodeCheckerService.js]: You might need to be a bit more thorough, as this is valid python:

`class        testClass   ():
  
  @classmethod
  def doAThing(cls, num):
    return num + 5
    
print testClass.doAThing(2)`

Comment (eyurko) [inline assets/tests/CodeCheckerService.js]: Hm, weird, it's not working. Lemme post a screenshot.

screen shot 2017-04-17 at 2 22 50 pm

Comment (lumig242) [inline assets/tests/CodeCheckerService.js]: So this function may check both the old/new style class definition right? (I'm a little confused by why you bold @classmethod).

Comment (lumig242) [inline assets/tests/CodeCheckerService.js]: Oh I see. Cause the (at) char is recognize by github lol.

Comment (lumig242) [inline assets/tests/CodeCheckerService.js]: What do you mean by "not working"? This snippet works at my side.

Comment (lumig242) [inline assets/tests/CodeCheckerService.js]: Just push the new changes

**Pull request merged**